src/components/DetailItems/CreatedAtDetailItem.tsx

```diff
@@ -0,0 +1,35 @@
+import type { FC } from 'react';
+import { ConsoleTimestamp } from 'src/components/ConsoleTimestamp/ConsoleTimestamp';
+import { DetailsItem } from 'src/modules/Providers/utils/components/DetailsPage/DetailItem';
+import { useForkliftTranslation } from 'src/utils/i18n';
+
+import { getCreatedAt } from '@utils/crds/common/selectors';
+
+import type { ResourceDetailsItemProps } from './utils/types';
+
+const CreatedAtDetailsItem: FC<ResourceDetailsItemProps> = ({
+  helpContent,
+  moreInfoLink,
+  resource,
+}) => {
+  const { t } = useForkliftTranslation();
+
+  const defaultMoreInfoLink = 'https://kubernetes.io/docs/reference/using-api/api-concepts';
```

Comment:
Please create, if it does not exist, a central place for all links and move it there